apps/server/src/modules/file-storage-client/interfaces/file-request-info.ts

```diff
@@ -0,0 +1,9 @@
+import { EntityId } from '@shared/domain';
+import { FileRecordParamsParentTypeEnum } from '../fileStorageApi/v3';
+
+export interface FileRequestInfo {
+	jwt: string; // todo replace with existing type
+	schoolId: EntityId;
+	parentType: FileRecordParamsParentTypeEnum;
```

Comment:
Like mentioned in another file: Shouldn't we use FileRecordParentType from filerecord.entity.ts here?